hr_attendance_report_theoretical_time/reports/hr_attendance_theoretical_time_report_views.xml

```diff
@@ -65,7 +71,7 @@
         <field name="res_model">hr.attendance.theoretical.time.report</field>
         <field
             name="context"
-        >{'search_default_previous_month': 1, 'search_default_current_month': 1, 'search_default_my': 1}</field>
+        >{'search_default_previous_month': 1, 'search_default_current_month': 1, 'search_default_my': 1, 'search_default_active_employees': 1}</field>
```

Comment:
I don't consider this filter should be active by default. You must seen all the attendances, being from archived employees or not.